simulator/infra/ports/module.h

```diff
@@ -21,17 +23,22 @@ class Module : public Log
     template<typename T>
     auto make_write_port( std::string key, uint32 bandwidth) 
     {
+        topology_write_ports.push_back(key);
```

Comment:
Please add a space after "(".
Can we use "emplace_back" here to make less string copies?